app/build.gradle

```diff
@@ -43,7 +43,7 @@ dependencies {
     testImplementation "org.testcontainers:junit-jupiter:${testcontainers_version}"
     testImplementation "org.testcontainers:postgresql:${testcontainers_version}"
     //This version is compatible with current spring version: 5.3.8
```

Comment:
This comment doesn't seem up-to-date since https://github.com/department-of-veterans-affairs/abd-vro/blob/0a79c7fe54af923e87a2c0adf2d6b1bf835cf825/gradle.properties#L34